tools/make_dictionary_file.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python2.7
+
+import sys
+"""
+Makes dictionary file from mozc
```

Comment:
s/mozc/Mozc dictionary/ would be better.

Also, can you add command line example?